pkg/rancher-desktop/components/Snapshots.vue

```diff
@@ -60,6 +75,13 @@ export default Vue.extend<Data, Methods, Computed, never>({
 
 <template>
   <div class="snapshots">
+    <Banner
```

Comment:
Can we make this banner closable by the user? I don't think this is a requirement, but I thought it would be nice to have the ability to dismiss the banner while testing this change.

Reply:
+1